test/Microsoft.Windows.ComputeVirtualization.Tests/project.json

```diff
@@ -10,11 +10,11 @@
     "debugType": "portable"
   },
   "runtimes": {
-    "win10-x64": {}
+    "win81-x64": {}
```

Comment:
Lets just file a task then. We can leave it open with a reminder to eventually look at it again.